src/app.jsx

```diff
@@ -62,6 +62,35 @@ class Application extends React.Component {
         });
     }
 
+    updateContainerStats (newContainerStats) {
+        this.setState({
+            containersStats: newContainerStats
+        });
+    }
```

Comment:
This is just being used once, and a wrapper around a simple function call. Please eliminate this function, to make the caller easier to read. Every React developer knows what `setState()` does, but when seeing `updateContainerStats()` in code, it's not immediately clear what this does. Also, the "container" vs. "containers" inconsistencies really don't help :)